docs/system/container-metrics.md

```diff
@@ -0,0 +1,104 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Container
+--->
+
+# Semantic Conventions for Container Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+## Container Metrics
+
+### Metric: `container.cpu.time`
+
+This metric is optional.
+
+<!-- semconv metric.container.cpu.time(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.cpu.time` | Counter | `s` | Total CPU time consumed [1] |
+
+**[1]:** Total CPU time consumed by the specific container on all available CPU cores
+<!-- endsemconv -->
+
+<!-- semconv metric.container.cpu.time(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| [`container.cpu.state`](../attributes-registry/container.md) | string | The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels. | `user`; `kernel` | Opt-In |
+
+`container.cpu.state` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+
+| Value  | Description |
+|---|---|
+| `user` | When tasks of the cgroup are in user mode (Linux). When all container processes are in user mode (Windows). |
+| `system` | When CPU is used by the system (host OS) |
+| `kernel` | When tasks of the cgroup are in kernel mode (Linux). When all container processes are in kernel mode (Windows). |
+<!-- endsemconv -->
+
+### Metric: `container.memory.usage`
+
+This metric is optional.
+
+<!-- semconv metric.container.memory.usage(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.memory.usage` | Counter | `By` | Memory usage of the container. [1] |
+
+**[1]:** Memory usage of the container.
+<!-- endsemconv -->
+
+<!-- semconv metric.container.memory.usage(full) -->
+<!-- endsemconv -->
+
+### Metric: `container.disk.io`
+
+This metric is optional.
+
+<!-- semconv metric.container.disk.io(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.disk.io` | Counter | `By` | Disk bytes for the container. [1] |
+
+**[1]:** The total number of bytes read/written successfully (aggregated from all disks).
+<!-- endsemconv -->
+
+<!-- semconv metric.container.disk.io(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| [`disk.io.direction`](../attributes-registry/disk.md) | string | The disk IO operation direction. | `read` | Recommended |
+| `system.device` | string | The device identifier | `(identifier)` | Recommended |
+
+`disk.io.direction` MUST be one of the following:
```

Comment:
can you please set `allow_custom_values: true` on this attribute in the registry? 
The context: https://github.com/open-telemetry/build-tools/issues/294

Reply:
I don't think it's a good idea to change [this](https://github.com/open-telemetry/semantic-conventions/blob/0d963c4016d96972bf0bfa51b23074d05bea8ec9/model/registry/disk.yaml#L10) as part of this PR, right?

This attribute is used by other metrics as well, so it would be better to change it separately once https://github.com/open-telemetry/build-tools/issues/294 is discussed?

Reply:
I'm fine keeping it since it does not change status quo. At the same time, `allow_custom_values: true` does not mean anything from the spec perspective (i.e. it's always ok to add new enum members).